changelogs/unreleased/2411-ashish-amarnath

```diff
@@ -0,0 +1 @@
+during backup deletion also delete CSI volumesnapshots that were created as a part of the backup
```

Comment:
I just noticed this changelog is the same as for the PR #2411.

Reply:
Actually, this PR is missing a change log. The change log that you are seeing here is bc this branch was cut from the PR #2411 branch. Will add a changelog